packages/babel-types/src/definitions/misc.js

```diff
@@ -19,3 +23,12 @@ defineType("Placeholder", {
     },
   },
 });
+
+defineType("V8IntrinsicIdentifier", {
```

Comment:
I just realized that we should add this to the allowed `callee`s of `CallExpression`